sale_pricelist_global_rule/models/product_pricelist.py

```diff
@@ -0,0 +1,320 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    def _compute_price_rule_get_items(
+        self, products_qty_partner, date, uom_id, prod_tmpl_ids, prod_ids, categ_ids
+    ):
+        items = super()._compute_price_rule_get_items(
+            products_qty_partner, date, uom_id, prod_tmpl_ids, prod_ids, categ_ids
+        )
+        # ignore new global rules on Odoo standard
```

Comment:
Why do you need this if you are cleaning the values on create/write?

Reply:
This is needed because Odoo does not evaluate the `applied_on` field; it only checks if `product_tmpl_id` or `categ_id` is NULL. In this case, both conditions are True, so the rules are considered. To discard these lines, this step is necessary.
https://github.com/odoo/odoo/blob/3a28e5b0adbb36bdb1155a6854cdfbe4e7f9b187/addons/product/models/product_pricelist.py#L106-L108